app/models/invoice.rb

```diff
@@ -442,21 +442,20 @@ def charge!(automatic = true)
     ensure_payable_state!
 
     unless chargeable?
-      logger.info "Not charging invoice ID #{self.id} (#{reason_cannot_charge})"
+      logger.info "Not charging invoice #{id} (#{reason_cannot_charge})"
       cancel! unless positive?
       return
     end
 
     if buyer_account.charge!(cost, :invoice => self)
-      provider.billing_strategy.try!(:info , "Charging invoice for #{buyer.name} for period #{period}", buyer)
+      provider.billing_strategy&.info("Invoice #{id} for period #{period} was charged, marking as paid", buyer)
```

Comment:
with original log message we could filter for "Charging invoice for <buyer>" and get both pass/error messages. Now strings are veru different and will require a more complicated query.

Reply:
Hm, they were actually different :) "Charging invoice for" vs "Charging for invoice for" :grimacing: 

I think now we can search for "Invoice 123" and it will actually return more information than before, WDYT?

Reply:
Maybe change to "Charging <buyer.id> for invoice <invoice id>". So that we can search easily both ways.

Reply:
here I prefer to say "was charged" as it is something that has already happened.

How about I replace all occurrences of `"invoice #{id}"` with `" invoice #{id} (buyer #{buyer_account_id})"`

Reply:
My point was to make searching for everything about a buyer with a simple query. Or everything about an invoice with a simple query. So your suggestion makes sense.

Reply:
Done!